ShaderView/src/main/java/com/appspell/shaderview/gl/params/ShaderParams.kt

```diff
@@ -12,6 +14,12 @@ interface ShaderParams {
     fun updateValue(paramName: String, value: Boolean)
     fun updateValue(paramName: String, value: FloatArray)
     fun updateValue(paramName: String, value: IntArray)
+    fun updateValue(
```

Comment:
`textureSlot` has already been set during initialization. I don't think we should change it updating values.

There are similar changes in this file in `main` branch https://github.com/appspell/ShaderView/blob/main/ShaderView/src/main/java/com/appspell/shaderview/gl/params/ShaderParams.kt